sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/firestore/it/BaseFirestoreIT.java

```diff
@@ -92,12 +92,12 @@ abstract class BaseFirestoreIT {
           .build();
 
   protected static String project;
-  protected GcpOptions options;
+  protected GcpOptions gcpOptions;
```

Comment:
nit: Do we need this anymore? Or can we just extract the project below?

Reply:
Removed.